src/main/java/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import service.ReaderService;
+
+public class ReaderServiceImpl implements ReaderService {
+
+    @Override
+    public List<String> read(String path) {
+        List<String> stringList = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(path))) {
+            br.readLine();
+            String line;
+            while ((line = br.readLine()) != null) {
+                stringList.add(line);
+            }
+            System.out.println("File read!");
```

Comment:
I dont think we need to print any info in console, lets remove this (we will use logger for this purpose later). Fix in all places